modules/rostests/apitests/crt/crtdata.c

```diff
@@ -0,0 +1,640 @@
+/*
+* PROJECT:         ReactOS CRT API tests
+* LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+* PURPOSE:         Tests for crt data exports
+* COPYRIGHT:       Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
+*/
+
+#include <apitest.h>
+#include <apitest_guard.h>
+
+#define WIN32_NO_STATUS
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <locale.h>
+#include <mbctype.h>
+
+static OSVERSIONINFOW s_osvi;
+extern char __ImageBase;
+static void* s_ImageEnd;
+
+int IsLocalSymbol(void* Address)
+{
+    return ((Address >= (void*)&__ImageBase) && (Address <= s_ImageEnd));
+}
+
+#ifndef TEST_STATIC
+#define test_is_local_symbol(addr, is) ok_int(IsLocalSymbol(addr), (is))
+#else
+#define test_is_local_symbol(addr, is)
+#endif
+
+void Test___argc()
```

Comment:
```suggestion
void Test___argc(void)
```

Reply:
Also if these test functions are just to be locally used in this file, they might become static?

Reply:
What exactly was this fixing? I'm asking, because surely, this is a significant fix, I just can't seem to remember why.

Reply:
But thanks for pointing out every single instance. Otherwise I might have missed one. These sneaky little bugs are hard to spot.

Reply:
Done